android/app/src/main/java/stytch/rn/test/newarchitecture/MainApplicationReactNativeHost.java

```diff
@@ -19,7 +19,7 @@
 import com.facebook.react.fabric.FabricJSIModuleProvider;
 import com.facebook.react.fabric.ReactNativeConfig;
 import com.facebook.react.uimanager.ViewManagerRegistry;
-import com.stytchdemo.BuildConfig;
+import stytch.rn.test.BuildConfig;
 import com.stytchdemo.newarchitecture.components.MainComponentsRegistry;
```

Comment:
Why are these different prefixes?

Reply:
I found that stytch.rn.test is where the BuildConfig is generated after building the app, and com.stytchdemo doesn't contain it. Not too sure why this is, I'll try to investigate if there's something we can do to change it